README.rst

```diff
@@ -270,6 +270,21 @@ Config settings
                 root /www/resource_cache;
             }
 
+6.  If you are not serving your archived resources through a webserver, you can upload them to ``s3filestore``. 
```

Comment:
say

Alternatively, you can upload archived resources to cloud storage. In this case, make sure that you have ...